example/lib/interactive_features_example.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:mapbox_maps_flutter/mapbox_maps_flutter.dart';
+import 'example.dart';
+
+class InteractiveFeaturesExample extends StatefulWidget implements Example {
+  @override
+  final Widget leading = const Icon(Icons.map);
+  @override
+  final String title = 'Interactive Features';
+  @override
+  final String? subtitle = 'Tap a Buildings to highlight it or a POI to hide it';
+
+  @override
+  State<StatefulWidget> createState() => InteractiveFeaturesState();
+}
+
+class InteractiveFeaturesState extends State<InteractiveFeaturesExample> {
+  InteractiveFeaturesState();
+  MapboxMap? mapboxMap;
+
+  _onMapCreated(MapboxMap mapboxMap) {
+    this.mapboxMap = mapboxMap;
+    mapboxMap.style;
+
+    /// Define interactions for 3D Buildings
+
+    // Define a tap interaction targeting the Buildings featureset in the Standard style
+    var tapInteraction = TapInteraction(Featureset.standardBuildings());
```

Comment:
On iOS/Android the Interaction itself consumes the callback upon construction, but in Flutter the callback is supplied into `addInteraction` method. Is it some limitation?